src/screens/PlrStaking/PlrStaking.tsx

```diff
@@ -286,6 +310,28 @@ const PlrStaking = () => {
         stakedPercentage={stakedPercentage}
         stakers={stakers}
         apy={stakingApy}
+        renderStakedInfo={() => {
```

Comment:
can we create a seperate function like `stakedInfoWrapper` or something like that and use here.